src/msg.c

```diff
@@ -1403,10 +1403,13 @@ static void start_timeout(struct l_timeout *timeout, void *user_data)
 static void append_on_update_list(void *data, void *user_data)
 {
 	const struct session *session = user_data;
+	knot_msg_schema *schema_found;
```

Comment:
About commit message ab14e2e73504ec9050da2978d9a6b50919948c5a:
Verify if sensor id exists in schema in update

This patch verifies if the thing has the sensor id in schema before sending the command update.

Or something like that.